<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>model_choose</class>
 <widget class="QMainWindow" name="model_choose">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>675</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>model-choose-mainwindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget_6" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1060</width>
      <height>621</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>model-choose-box</string>
    </property>
    <widget class="QLabel" name="video_label">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>0</y>
       <width>840</width>
       <height>520</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QWidget" name="widget_5" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>220</width>
       <height>620</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string>model-choose-oprate</string>
     </property>
     <widget class="QPushButton" name="search_model">
      <property name="geometry">
       <rect>
        <x>19</x>
        <y>310</y>
        <width>181</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="whatsThis">
       <string>btn</string>
      </property>
      <property name="text">
       <string>搜索可用模型</string>
      </property>
     </widget>
     <widget class="QWidget" name="widget_4" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>230</y>
        <width>221</width>
        <height>50</height>
       </rect>
      </property>
      <property name="whatsThis">
       <string>choose-box</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="whatsThis">
          <string>label-model</string>
         </property>
         <property name="text">
          <string>模 型 选 择：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="model_choose_box">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>19</x>
        <y>550</y>
        <width>181</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="whatsThis">
       <string>btn</string>
      </property>
      <property name="text">
       <string>退出</string>
      </property>
     </widget>
     <widget class="QPushButton" name="model_test">
      <property name="geometry">
       <rect>
        <x>19</x>
        <y>390</y>
        <width>181</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="whatsThis">
       <string>btn</string>
      </property>
      <property name="text">
       <string>模型测试</string>
      </property>
     </widget>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>90</y>
        <width>221</width>
        <height>50</height>
       </rect>
      </property>
      <property name="whatsThis">
       <string>input-box</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="whatsThis">
          <string>label-NMS</string>
         </property>
         <property name="text">
          <string>NMS 阈  值：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="NSM_edit">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>19</x>
        <y>470</y>
        <width>181</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="whatsThis">
       <string>btn</string>
      </property>
      <property name="text">
       <string>保存设置</string>
      </property>
     </widget>
     <widget class="QWidget" name="widget_3" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>160</y>
        <width>221</width>
        <height>50</height>
       </rect>
      </property>
      <property name="whatsThis">
       <string>input-box</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="whatsThis">
          <string>label-threshod</string>
         </property>
         <property name="text">
          <string>置信度阈值:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="confi_edit">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>221</width>
        <height>50</height>
       </rect>
      </property>
      <property name="whatsThis">
       <string>input-box</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="whatsThis">
          <string>label-foclenth</string>
         </property>
         <property name="text">
          <string>焦距  (MM)：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="foclenth_edit">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="whatsThis">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>520</y>
       <width>840</width>
       <height>100</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="whatsThis">
      <string>attention-label</string>
     </property>
     <property name="text">
      <string>注意事项:摄像头焦距需要尽可能接近真实参数、模型参数调整之后需要保存配置才能在系统生效、模型测试仅仅提供效果显示</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1080</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
